cime_config/usermods_dirs/B127ka/user_nl_cism

```diff
@@ -0,0 +1,20 @@
+!-----------------------------------------------------------------------
```

Comment:
This effectively empty user_nl file should be removed.

Reply:
Removed